controller/pokemon.go

```diff
@@ -0,0 +1,29 @@
+package controller
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/MoraAlex/academy-go-q32021/services"
```

Comment:
Not a big deal, but add a blank line here to separate the local import from the third party.